examples/step-21/doc/intro.dox

```diff
@@ -250,6 +250,9 @@ cell term to get an equation as follows:
   (S^n,\sigma)_\Omega +
   \triangle t \sum_K  \left(F(S^n) q^{n+1}, \sigma\right)_K.
 @f}
+In order to keep track of the current value of time and the time step in the code,
+an object of type DiscreteTime is used. This class encapsulates many complexities
```

Comment:
Maybe more as general advice for writing English language: Try to avoid using the passive voice ("is used"). There are just so many ways in which passive voice can be misunderstood. Just say "we use an object of type DiscreteTime"

Reply:
Thank you for the advice, I changed some of the sentences to use the active voice.